src/Uno.Foundation/FoundationFeatureConfiguration.cs

```diff
@@ -38,8 +38,9 @@ public static class Runtime
 		/// </summary>
 		public static bool RethrowNativeExceptions
 		{
-			get => WebAssembly.Runtime.RethrowNativeExceptions;
-			set => WebAssembly.Runtime.RethrowNativeExceptions = value;
+			// Obsolete, remove in next major
```

Comment:
nit: We could add EditorBrowsable(Never) for current release

Reply:
Good idea, will add to https://github.com/unoplatform/uno/pull/18135.